internal/services/batch/batch_pool_resource_test.go

```diff
@@ -740,6 +740,24 @@ func TestAccBatchPool_interNodeCommunicationWithTaskSchedulingPolicy(t *testing.
 	})
 }
 
+func TestAccBatchPool_securityProfileWithUEFISettings(t *testing.T) {
```

Comment:
could we add additional steps to this test to test updating the properties in the block and to test adding/removing the block?

Reply:
Sure, will include creating and removal

Reply:
Plus I will mark this block and every property as forceNew